### DISCUSSION TITLE
seata启动可以 但是就是不会出现再nacos的注册列表

### DISCUSSION THREAD

**Discussion by csky82** [apache/incubator-seata#4770] [Q&A] [answered]
版本1.5.1 可以启动 我用的是挂载配置文件的方式 docker部署启动的 启动可以启动 但是好像根本没有读配置文件 之前1.4.2都还是好的 nacos注册列表里也没有seata 其它服务一直报 no available service found in cluster 'seata-server', please make sure registry config correct and keep your seata server running  很奇怪 到底哪里出了问题 ？
下面是我的registry.conf配置文件
==================================================
registry {
  # file 、nacos 、eureka、redis、zk、consul、etcd3、sofa
  type = "nacos"
  nacos {
    application = "seata-server"
    serverAddr = "XXX.XXX.XXX.XXX:XXXX"
    namespace = ""
    group = "SEATA-GROUP"
    username = "nacos"
    cluster = "seata-server"
    password = "123456"
    weight = 1
  }
}
 
config {
  # file、nacos 、apollo、zk、consul、etcd3
  type = "nacos"
  nacos {
    serverAddr = "XXX.XXX.XXX.XXX:XXXX"
    namespace = "XXXXXXX"
    group = "test"
    cluster = "seata-server"
    username = "nacos"
    password = "123456"
    dataId = "seataServer.properties"
  }
}

**Comment by wangzhencq** [comment]
我用的1.5.2  也遇到同样的问题，用1.4.2 就没有这样的问题

**Comment by slievrly** [comment, marked as answer]
@wangzhencq @csky82  1.5.x 版本的seata-server 已经改造成springboot，不再读取conf配置文件，请挂载这个配置文件：https://github.com/seata/seata/blob/develop/server/src/main/resources/application.yml

**Reply by csky82** [reply]
请问还是使用SEATA_CONFIG_NAME指定配置文件吗？这个好像也不起作用

**Reply by slievrly** [reply]
SEATA_CONFIG_NAME 是针对conf配置文件，1.5.1 seata-server读取的是yml配置文件，需要直接-v挂载，环境变量对于yml不起作用了。

**Reply by stheart** [reply]
您好，能麻烦更新下V1.5.2官网文档说明吗，我查这个问题也是花了好久时间
